### PR TITLE
Feature/how develop document

### PR DESCRIPTION
**Pull request by feloxx**
- add how develop document

### CODE REVIEW

File: zh_cn/docs/开发者/开发入门/如何给VNote开发文档.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+![](vx_images/151165171356026.png)
+
+注意，此时是提交到了自己的 Github 中，需要到页面上去发起 PR 合并申请。
+
+![](vx_images/584175538751262.png)
+
+发起 PR 合并申请的时候，**千万！ 千万！ 注意选对分支**
+
+
+```
```
Comment (tamlok) [inline zh_cn/docs/开发者/开发入门/如何给VNote开发文档.md:81]: You can use:

```
::: alert-info
分支请选择gh-pages
:::
```

File: zh_cn/docs/开发者/vx.json [hunk 2]
```diff
@@ -13,6 +13,9 @@
         }
     ],
     "folders": [
+        {
+            "name": "开发入门"
```
Comment (tamlok) [inline zh_cn/docs/开发者/vx.json:17]: Unnecessary to create a new folder. Just put it under the parent folder.

Comment (feloxx) [reply zh_cn/docs/开发者/vx.json:17]: I'm going to write a series of documents.

File: zh_cn/docs/开发者/开发入门/如何给VNote开发文档.md (hunk 1 above)
Comment (feloxx) [reply zh_cn/docs/开发者/开发入门/如何给VNote开发文档.md:81]: resolved

**Pull request merged**